package.json

```diff
@@ -30,13 +30,15 @@
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && cd _book && git init && git commit --allow-empty -m 'update book' && git fetch https://github.com/airbnb/react-sketchapp gh-pages && git checkout -b gh-pages && git add . && git commit -am 'update book' && git push https://github.com/airbnb/react-sketchapp gh-pages --force",
     "lint-staged": "lint-staged",
-    "prepublishOnly": "npm run clean && npm run test:ci && npm run build",
+    "prepublishOnly": "npm run clean && npm run test:ci && npm run install:e2e && npm run build",
```

Comment:
`npm run install:e2e` triggers the `__tests__/examples` `postinstall` script. Not sure if this should be left in, it'll fail the prepublish script if the examples `skpm-build` fails, but not sure if it'll work in CI.